### DISCUSSION TITLE
Seq Alert - Conditional Notification Channel

### DISCUSSION THREAD

**Discussion by ryanbuening** [datalust/seq-tickets#2003] [Help] [answered]
We have Seq alerts setup with a Notification Channel (in our case, Microsoft Teams). We have two channels in Teams: a `Production Errors` channel and a `Test Errors` channel. 

Currently, we have two Seq alerts setup so that we can identify and send to the correct Notification Channel based upon the Signal (e.g. Errors in Test go to the `Test Errors` channel. The `where` clause is the same though, so it is duplicated between both Seq alerts.

Is there a way to setup one alert, but conditionally choose the Notification Channel based upon the event? For example, if the event came from Test, it would only go to the `Test Errors` Notification Channel? This would cut down on the duplicate alerts we are currently setting up.

**Comment by nblumhardt** [comment, marked as answer]
Thanks for the suggestion, @ryanbuening.

It's not possible right now, though it's something we've talked about in the past.

I'll convert this to an issue so that we can discuss and triage it when we start planning our next release (we're deep in 2024.1 currently, but expect it to be done early next year 🤞).
